blog/2025-10-01-september-releases.md

```diff
@@ -0,0 +1,68 @@
+---
+title: September 2025 Releases
+slug: 2025-10-01-september-releases
+authors: [kenodegard]
+tags: [announcement, conda, conda-build]
+description: |
+  conda 25.9.0 and conda-build 25.9.0 have been released! 🎉
+image: img/blog/2025-10-01-september-releases/banner.png
+---
+
+The September 2025 releases included updates to conda and conda-build! 🎉 Both have been released to `defaults` and `conda-forge` channels.
+
+<!-- truncate -->
+
+## Changes in conda [25.9.0](https://github.com/conda/conda/releases/tag/25.9.0)
+
+:::warning Special Announcement
+
+Following feedback from conda users last year about the pre-configuration of the conda code base to favor channels from Anaconda Inc, we've [finished removing](https://github.com/conda/conda/issues/14178) hardcoding Anaconda's channels as the default set of channels in the conda source code.
+
+It is now up to providers of the conda distributions, such as [miniforge](https://github.com/conda-forge/miniforge) or Anaconda (including miniconda), to pre-configure their preferred channels, e.g. by running the necessary `conda config --set channels` command.
+
+:::
+
+To update `conda` to the latest version, run:
+
+```bash
+conda install -n base conda=25.9.0
+```
+
+**Notable Changes:**
+
+- Added two new health checks to `conda doctor`:
+    - Check for ill-formed pinned file.
+    - Check whether file locking is supported.
+- Prevent renaming an envrironmnet currently listed as the `default_activation_env`.
+- Added a new environment specifier, `cep-24`. This is a stricter version of the `environment.yml` format and can be used when creating an environment from a file, e.g., `conda env create --file FILE --environment-specifier cep-24`.
```

Comment:
I think it would be great to link to the CEP here.

```suggestion
- Added a new environment specifier, `cep-24`. This is a stricter version of the `environment.yml` format that enforces [specific requirements](https://conda.org/learn/ceps/cep-0024/) and can be used when creating an environment from a file, e.g., `conda env create --file FILE --environment-specifier cep-24`.
```